index.d.ts

```diff
@@ -241,13 +243,14 @@ declare module 'react-native-nfc-manager' {
       toHex: (i: any) => any;
       toPrintable: (i: any) => string;
     };
-    isType(record: NdefRecord, tnf: number, type: string): boolean;
+    isType(record: NdefRecord, tnf: TNF, type: string): boolean;
     stringify(data: number[], separator: string): string;
-    encodeMessage(records: NdefRecord[]): Uint8Array;
-    decodeMessage(bytes: any[] | Buffer): NdefRecord[];
```

Comment:
Removed buffer type since it is not declared on this project and neither on any dependent package, so it breaks typescript builds trying on downstream consumer projects 

![image](https://user-images.githubusercontent.com/11574849/86190070-c90efa80-bb19-11ea-8808-8ea974a122c2.png)